ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py

```diff
@@ -99,13 +99,13 @@ def __init__(self,
         self.storage_volume_size = 2 * self.root_volume_size
 
     # ---------------------------------------------------------------------
-    def _attach_volume(self, instance, volume, device=None):
+    def _attach_volume(self, volume, device=None):
```

Comment:
I'm missing the refactoring where you call ``_attach_volume`` (except the calls in this method of course)

Reply:
Fixed it now